lms/djangoapps/support/static/support/jsx/logged_in_user.jsx

```diff
@@ -37,7 +37,7 @@ function LoggedInUser({ userInformation, setErrorState, zendeskApiHost, submitFo
   let subjectElement;
   subjectElement = (<div>
     <label htmlFor="subject">{gettext('Subject')}</label>
-    <select className="form-control select-subject" id="subject">
+    <select className="form-control select-subject" id="subject" onChange={setDisplay} value={this.state.value}>
```

Comment:
Where is value being set? ie. what is this.state.value?

Reply:
this was a test thing, def a bad add.